dtfs-central-api/src/repositories/tfm-deals-repo/tfm-deal-cancellation.repo.findDealCancellationByDealId.test.ts

```diff
@@ -11,9 +11,12 @@ const mockDealObject = { tfm: { cancellation: mockDealCancellationObject } };
 
 describe('tfm-deals-cancellation-repo', () => {
   const findOneMock = jest.fn();
-  const updateOneMock = jest.fn();
   const getCollectionMock = jest.fn();
 
+  beforeAll(() => {
+    jest.useFakeTimers().setSystemTime(1727876437063);
```

Comment:
Why do we need to do this please?

Reply:
Oops - added in for the audit logs in update & delete but didn't remove when splitting the tests up